src/components/__tests__/MoneyRaised.test.js

```diff
@@ -108,8 +112,33 @@ describe('MoneyRaised component', () => {
       },
     })
 
-    expect(wrapper.find('span').text()).toEqual('$650,200.62')
+    expect(wrapper.find(Typography).first().text()).toEqual('$650,200.62')
     act(() => jest.advanceTimersByTime(30e3)) // 30 seconds
-    expect(wrapper.find('span').text()).toEqual('$650,201.62')
+    expect(wrapper.find(Typography).first().text()).toEqual('$650,201.62')
+  })
+
+  it('displays the popup when clicked on', () => {
+    const MoneyRaised = require('src/components/MoneyRaised').default
+    const defaultMockProps = getMockProps()
+
+    const wrapper = mount(<MoneyRaised {...defaultMockProps} />)
+    expect(wrapper.find(DashboardPopover).prop('open')).toBe(false)
+
+    wrapper.find(Button).first().simulate('click')
+    expect(wrapper.find(DashboardPopover).prop('open')).toBe(true)
+  })
+
+  it('popover onClose sets isPopoverOpen to false', () => {
```

Comment:
This one is really stumping me as it's failing. Would you mind taking a look @kmjennison, hope I'm not missing something obvious.